lab-jaren/test/read-files-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const expect = require('chai').expect;
+const readFiles = require('../lib/read-files.js');
+
+describe('testing module readFiles', function(){
+  describe('with a file that does not exist', function(){
+    it('should pass an error into the callback', function(done){
+      readFiles([`${__dirname}/../assets/does-not-exist.txt`], function(err, data){
+        //console.log(err);
+        expect(Boolean(err)).to.equal(true);
+        console.log(data);
+        done();
+      });
+    });
+  });
+
+  describe('with valid inputs', function(){
+    var inputFiles = [`${__dirname}/../assets/star.txt`,
+      `${__dirname}/../assets/wars.txt`,
+      `${__dirname}/../assets/ipsum.txt`,
+    ];
+    it('should pass an array of text data that maps from the file paths array', function(done){
+      var result = [ 'Well, that\'s the trick, isn\'t it? And it\'s going to cost you something extra. Ten thousand in advance. Ten thousand? We could almost buy our own ship for that! But who\'s going to fly it, kid! You? You bet I could. I\'m not such a bad pilot myself! We don\'t have to sit here and listen... We haven\'t that much with us. But we could pay you two thousand now, plus fifteen when we reach Alderaan. Seventeen, huh! Okay. You guys got yourself a ship. We\'ll leave as soon as you\'re ready. Docking bay Ninety-four. Ninety-four. Looks like somebody\'s beginning to take an interest in your handiwork. All right, we\'ll check it out.\n',
```

Comment:
Thomas and I had a hard time figuring out how to avoid copy/pasting ALL the text to test the proper outputs. In the end, Duncan helped him (and me too :) with a DRYer version of the test module. 

It's still NOT the dryest it can be, but it's still better-looking than copy/pasting text. You can look at Thomas's test module in his PR here:

[https://github.com/codefellows-seattle-javascript-401d14/lab-03-readfiles/pull/4/files](https://github.com/codefellows-seattle-javascript-401d14/lab-03-readfiles/pull/4/files)

I hope this helps!